apps/web/google_docs.talon

```diff
@@ -0,0 +1,326 @@
+browser.host: docs.google.com
+-
+
+tag(): user.line_commands
+
+file menu | menu file: key(ctrl-alt-f)
+open: key(cmd-o)
+[open] (revision | version) history [open]: key(cmd-alt-shift-h)
+print: key(cmd-p)
+save: key(cmd-s)
+edit menu | menu edit: key(ctrl-alt-e)
+paste without formatting: key(cmd-shift-v)
+# adding keyboard shortcuts here since user.find_and_replace doesn't work
```

Comment:
Please implement the `find_and_replace` tag instead of redefining these commands. You can see the actions that need to be implemented in [`tags/find_and_replace/find_and_replace.py`](https://github.com/knausj85/knausj_talon/blob/main/tags/find_and_replace/find_and_replace.py).